QubitServer/src/main/java/org/labrad/qubits/config/AdcBaseConfig.scala

```diff
@@ -0,0 +1,41 @@
+package org.labrad.qubits.config
+
+import java.util.Map
+
+import org.labrad.data.Request
+
+abstract class AdcBaseConfig(channelName: String, builProperties: Map[String, Long]) {
+
+  /**
+   * number of clock cycles to delay
+   */
+  protected var startDelay: Int = -1
```

Comment:
Actually, I just checked and this is not in fact used anywhere. The `startDelay` value is set directly on the fpga model, and this one never gets set or read (I tried removing the accessor methods and everything still compiles just fine). I'm going to remove these for now since they're not needed.